llvm/docs/DeveloperPolicy.rst

```diff
@@ -853,6 +853,57 @@ their patch with every possible configuration.
 * 3rd step: If Galina could not help you, please escalate to the
   `Infrastructure Working Group <mailto:iwg@llvm.org>`_.
 
+Guidelines for fuzzer-generated issues
+--------------------------------------
+
+Fuzzing is a valuable tool for finding compiler bugs, and the LLVM project
+welcomes fuzzer-generated test cases. However, some additional guidelines
+should be followed to make such reports maximally useful.
+
+Fuzzer-generated issues should indicate that they are such, either in the
+issue description, or (for organization members) by applying the
+``fuzzer-generated`` label. This helps us prioritize issues. The remaining
```

Comment:
Note currently folks are using `clang:fronted:fuzzer` and `compiler-rt:fuzzer` I don't have a strong preference here but I would be curious to know how folks feel about having one bucket Vs more fine grained ones.

Reply:
My preference is to have a single label that can be combined with others. At least for the sake of label length and total number of labels we have, which is overwhelming already.

Reply:
`compiler-rt:fuzzer` is for libfuzzer in compiler-rt, not for fuzzer-generated issues.